src/Model/RefundTypeInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Model;
+
+interface RefundTypeInterface
```

Comment:
Could we add `valid()` method here to take care of https://github.com/Sylius/RefundPlugin/pull/311/files#r652550575?